aips/aip-reduce-stake-lockup-duration.md

```diff
@@ -0,0 +1,59 @@
+---
+aip: 
+title: Reduce stake lockup duration to 14d and governance voting duration to 3d per community vote
+author: Tony (https://github.com/tony001242), Eric (eric@amnis.com)
+discussions-to (*optional): 
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Core
+created: 7/11/2024
+updated (*optional): <mm/dd/yyyy>
+requires (*optional): <mm/dd/yyyy>
+---
+
+# AIP-90 - Add Apple as a supported OIDC provider
+
+## Summary
+
+Aptos (APT), the layer 1 blockchain that we’re building upon, employs a Proof of Stake (PoS) consensus mechanism, wherein validators stake tokens to participate in network operations and governance.
+Currently, Aptos has a 30-day lock-up period for staked tokens, one of the longest among layer 1 protocols. This duration aims to ensure network stability and robust on-chain governance. However, feedback from various stakeholders suggests that reducing this period might offer significant benefits.
+As such, we’re proposing some possible implementation options for our community to consider. We conducted a survey within the community, and the winning option is to reduce the lock-up period to 14 days and the voting period to 3 days.
```

Comment:
should provide some background on why these two configurations is coupled with each other for people who does not have details on how governance works.